pkg/renderer/html_test.go

```diff
@@ -0,0 +1,62 @@
+package renderer
+
+import (
+	"bytes"
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+func TestRederingHTML(t *testing.T) {
+	tempDir := t.TempDir()
+	HTML.Configure("/assets", tempDir, map[string]string{"some_path/something.asset": "some_path/other_path/random.asset"})
+	setupHTMLFile(t, path.Join(tempDir, "test.html"))
+
+	htmlBuffer := bytes.NewBuffer([]byte(""))
```

Comment:
Eu acho que os testes estão sempre adicionando mais informação no mesmo buffer, o que pode ser um comportamento inesperado e levar a falso positivos or falso negativos. 

Por exemplo, se você adicionar um teste que espera um certo valor no resultado, mas esse valor na verdade foi adicionado por um caso anterior, o teste passaria, mesmo com o valor não sendo adicionado no caso esperado.

Acho que seria melhor criar um buffer por teste, assim cada case roda sem influência dos outros. 

Um outro ponto para o futuro seria rodar is testes em paralelo, o que exige chamar `t.Parallel()`, mas com um buffer compartilhado, esses cenários não poderiam rodar em paralelo.